tests/unit/kvmagent/test_vm_iothread_vq_mapping.py

```
@@ -0,0 +1,382 @@
+        'useVirtioSCSI': False,
+        'multiQueues': '16',
+        'ioThreads': 8,
+        'volumeUuid': 'volume-uuid',
+        'deviceId': 1,
+    }
+    values.update(kwargs)
+    return Volume(**values)
+
+
+class EmptyAddons(object):
+    attachedDataVolumes = []
```

Ruff (0.15.15)

[warning] Mutable default value for class attribute

(RUF012)